packages/runtime/src/next/components/tests/controls/page-control-prop-rendering.tsx

```diff
@@ -52,23 +98,21 @@ export async function testPageControlPropRendering<D extends ControlDefinition>(
   const controlData: DataType<D> | Data =
     value !== undefined ? (toData ? toData(value) : controlDefinition.toData(value)) : undefined
 
-  const TestComponentType = 'TestComponent'
+  const testComponentMeta = {
+    type: 'TestComponent',
+    label: 'Test Component',
+  }
+
```

Comment:
Here and below, some test restructuring/cleanup to incorporate a server-side rendering branch.